views/index.ejs

```diff
@@ -2,18 +2,15 @@
 <html>
   <head>
       <% include ./partials/head %>
+      <script src="http://code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
```

Comment:
Right now this is used in index.js indeed. If you edit that code you can remove it, for now I will keep it there. Btw note that it is not used for ajax. For ajax we use iron-ajax from socket.io (which should not be replaced, initial loading of the page is done by polymer, updating/scrolling on it can be done from socket.io)